README.md

```diff
@@ -17,8 +17,7 @@ Add the following in your `Package.swift` file
 ## 📘 Overview
 
 First, install [wkhtmltopdf](http://wkhtmltopdf.org/downloads.html). This
-library is tested on version 0.12.4. Your binary should be installed at
-`/usr/local/bin/wkhtmltopdf`; run it to ensure it and any dependencies are
+library is tested on version 0.12.4. Specify the location of `wkhtmltopdf` in the `Document` initialiser. The default is `/usr/local/bin/wkhtmltopdf`. Run it to ensure it and any dependencies are
```

Comment:
Nitpick: could you please add newlines before column 80 to maintain consistency?